File: OLSource1/v1/VS_officedev/codesnippet/CSharp/specific-security-considerations-for-office-solutions_2.cs
        private void TrustedCode()
        {
            Microsoft.Office.Interop.Outlook.MailItem mailItem1 =
                this.Application.CreateItem(
                Microsoft.Office.Interop.Outlook.OlItemType.olMailItem) as
                Microsoft.Office.Interop.Outlook.MailItem;
            mailItem1.To = "someone@example.com";
            MessageBox.Show(mailItem1.To);
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/specific-security-considerations-for-office-solutions_2.vb
    Private Sub TrustedCode()
        Dim mailItem1 As Microsoft.Office.Interop.Outlook.MailItem = _
            TryCast(Me.Application.CreateItem( _
            Microsoft.Office.Interop.Outlook.OlItemType.olMailItem),  _
            Microsoft.Office.Interop.Outlook.MailItem)
        mailItem1.To = "someone@example.com"
        MessageBox.Show(mailItem1.To)
    End Sub